### ISSUE TITLE
Consequtive returns inside if statement

### ISSUE THREAD

**Issue by forthrin** [rubocop/rubocop#15097]
### Before submitting

- [x] I have searched the existing issues and this hasn't been reported yet.
- [x] I am running the latest stable version of RuboCop.

### Issue type

False positive

### Cop name

Layout/EmptyLineAfterGuardClause

### Steps to reproduce the problem

```ruby
def foo(x)
  if x
    return 'one' if x == 1
    return 'two' if x == 2
    return 'three or more'
  end
  x
end

```

### Expected behavior

Not so much as a parking ticket

### Actual behavior

```ruby
    return if x == 2
    ^^^^^^^^^^^^^^^^
```

Would seem related to #15090 & friends

### RuboCop configuration

N/A

### RuboCop version

```shell
System Version: macOS 26.3.1 (25D2128)
Model Identifier: MacBookAir10,1
ruby 4.0.2 (2026-03-17 revision d3da9fec82) +PRISM [arm64-darwin25]
rubocop     2026-04-09 6d2df6446
```

**Comment by koic** [comment]
This is different from #15090. A guard condition and a simple `return` are not the same, and this behavior is expected.

**Comment by forthrin** [comment]
So 100 `return if` can spoon, but the final unconditional `return` must sleep on the floor? Guess that sort of makes sense.
